packages/c/srv/src/srv.c

```diff
@@ -139,7 +139,7 @@ int run_srv_daemon_side_multi(srv_params_t *params) {
       len = res;
     }
 
-    if (&control_side.transformer != NULL) {
+    if (control_side.transformer != NULL) {
```

Comment:
transformer is already a pointer, this was grabbing the address of address...

Reply:
This is exactly why I have -Werror enabled in the CI, to ensure that no matter what tooling we use across the team we catch things like this...